### ISSUE TITLE
`deno fmt` ignores `.prettierrc.json` file

### ISSUE THREAD

**Issue by Alhadis** [denoland/deno#3276]
There doesn't appear to be a way to configure `deno fmt`, and its `--help` output mentions nothing about config files (apart from `tsconfig.json`).

I ran `deno fmt .` from a directory with a `.prettierrc.json` file containing:

~~~json
{
	"tabWidth": 4,
	"useTabs": true,
	"semi": true,
	"singleQuote": false,
	"trailingComma": "es5",
	"bracketSpacing": false,
	"arrowParens": "avoid"
}
~~~

Which `deno` completely ignored.

**Comment by bartlomieju** [comment]
This issue is kinda duplicate of #3033.

**Comment by axetroy** [comment]
Try to load the config file in order

- .prettierrc.json
- .prettierrc.yaml
- .prettierrc.yml
- .prettierrc.js
- prettier.config.js
- .prettierrc.toml

before run format

https://github.com/denoland/deno/blob/b3baab6d1479abbfd2e534fe05f5e146c2d5b1a2/std/prettier/main.ts#L334-L349

Reference: https://prettier.io/docs/en/configuration.html

PR is welcome

**Comment by axetroy** [comment]
after #3314 land. I will do this.
